Makefile

```diff
@@ -11,38 +11,40 @@ endif
 
 NOOP = true
 ifeq ($(PAASTA_ENV),YELP)
-	# This index must match the Ubuntu codename in the dockerfile.
-	DOCKER_PIP_INDEX_URL ?= http://169.254.255.254:20641/bionic/simple/
+	export PIP_INDEX_URL ?= http://169.254.255.254:20641/jammy/simple/
```

Comment:
we removed a comment here, but we still need to handle what the comment was referring to - i'm assuming that if we do `make itest_bionic` locally it will fail since it'll pull down jammy wheels in a bionic container

Reply:
(i left a comment in the PR that added this index url stuff for a way to handle multiple codenames)